pkg/loader/model.go

```diff
@@ -37,6 +35,13 @@ const (
 	DeleteDMLType  DMLType = 3
 )
 
+// Destination database type can be Mysql/Tidb or Oracle
+const (
+	DBTypeUnknown = iota
+	MysqlDB
+	OracleDB
+)
```

Comment:
```suggestion
type DBType int
const (
	DBTypeUnknown DBType = iota
	MysqlDB
	OracleDB
)
```
Always use `DBType` instead of `int`